src/components/Results.tsx

```diff
@@ -26,6 +26,18 @@ const Results: React.FC<{
       <button onClick={resetQuiz} className="results-btn">
         Play again?
       </button>
+
+      <a
+        className="results-text"
+        href={
+          "http://twitter.com/intent/tweet?text=I just scored " +
+          ((Math.floor(points) / totalPoints) * 100).toString() +
+          "% on the Learn to developerquiz.org - try it yourself?" +
+          "&hashtags=freecodecamp"
+        }
+      >
```

Comment:
When I tested this out, it initially went to a 500 error. Maybe there was some sort of syntax error.

<img width="709" alt="Screen Shot 2022-10-15 at 1 51 56 PM" src="https://user-images.githubusercontent.com/67210629/196010781-4cf08b15-8d05-4d6c-a136-17dde7c3e5b6.png">

I refactored this to be a template literal and it is working now. Also, this will be cleaner to read. I added a condition to only show this tweet button if they score 70% or higher. We don't want them tweeting their score if it is 10 or 20%.

```
  const totalPercentageCorrect = (Math.floor(points) / totalPoints) * 100;
  const tweetMessage = `http://twitter.com/intent/tweet?text=I just scored ${totalPercentageCorrect}%25 on developerquiz.org. Wanna try it for yourself?&hashtags=freecodecamp`;

   {totalPercentageCorrect >= 70 && (
        <a
          target="_blank"
          rel="noreferrer"
          className="results-text"
          href={tweetMessage}
        >
          <i className="fab fa-twitter" /> Tweet your quiz score
        </a>
      )}
```

Reply:
Thank you for fixing it. And yeah that makes sense.